packages/turf-line-chunk/.npmignore

```diff
@@ -0,0 +1,2 @@
+test
+coverage
```

Comment:
You can remove `.npmignore` since this is covered by `files:[*]` in the `package.json`